views/stats.ejs

```diff
@@ -2138,7 +2160,19 @@ twitterDescription += '. Click to view more stats!'
                         <div class="minion-category-header"><div class="minion-category-icon"><div class="item-icon custom-icon" style="background-image: url(/head/93c8aa3fde295fa9f9c27f734bdbab11d33a2e43e855accd7465352377413b)"></div></div><span>pet milestones</span></div>
                         <p class="stat-raw-values">
                             <% for(const key in calculated.misc.milestones){ %>
-                            <span class="stat-name"><%= helper.capitalizeFirstLetter(key.split("_").join(" ")); %>: </span><span class="stat-value"><%= calculated.misc.milestones[key].toLocaleString() %></span><br>
+                            
+                                <%var percentage = {};%>
+                                <%for(const pet of pet_milestones){%>
+                                    <%if(pet.id == key){ %>
+                                        <%for(var i = 0; i < pet.milestones.length; i++){%>
```

Comment:
```suggestion
                                        <%for(let i = 0; i < pet.milestones.length; i++){%>
```
Use `let` or `const` instead of `var`